dev/fattest.simplicity/src/componenttest/custom/junit/runner/RepeatTestFilter.java

```diff
@@ -147,12 +150,12 @@ public static String getRepeatActionsAsString() {
     }
 
     /**
-     * Is the repeat action active?
+     * Is the repeat action currently active?
      *
      * @param  action The repeat action to check.
-     * @return        True if the repeat action is active.
+     * @return True if the repeat action is active.
      */
     public static boolean isRepeatActionActive(String action) {
-        return REPEAT_ACTION_STACK.contains(action);
+        return REPEAT_ACTION_STACK.peek() == null ? false : REPEAT_ACTION_STACK.peek().equals(action);
```

Comment:
I can't say if this is correct or not... you need to ask around more widely

Reply:
Removed for now.